FreeRTOS/FreeRTOS
# Add Vectored Interrupt Support To SiFive RISC-V Demo

Description
-----------
Updates the SiFive HiFive demo to allow for both use of vectored interrupts and a standard interrupt handler.

Test Steps
-----------
Confirmed with Guarav (who ran the demo) that these changes worked.

Related Issue
-----------
No related issue. Updating IAR demo to match GCC demo functionality. 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
